photon-lib/src/main/native/cpp/photon/PhotonCamera.cpp

```diff
@@ -39,26 +41,79 @@
 #include <wpi/json.h>
 
 #include "PhotonVersion.h"
+#include "opencv2/core/utility.hpp"
 #include "photon/dataflow/structures/Packet.h"
 
-inline constexpr std::string_view bfw =
-    "\n\n\n\n"
-    ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
-    ">>> !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n"
-    ">>>                                          \n"
-    ">>> You are running an incompatible version  \n"
-    ">>> of PhotonVision on your coprocessor!     \n"
-    ">>>                                          \n"
-    ">>> This is neither tested nor supported.    \n"
-    ">>> You MUST update PhotonVision,            \n"
-    ">>> PhotonLib, or both.                      \n"
-    ">>>                                          \n"
-    ">>> Your code will now crash.                \n"
-    ">>> We hope your day gets better.            \n"
-    ">>>                                          \n"
-    ">>> !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n"
-    ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
-    "\n\n";
+inline void verifyDependencies() {
+  if (!(GetWPILibVersion() == photon::PhotonVersion::wpilibTargetVersion)) {
+    std::string bfw =
```

Comment:
Consider using fmtlib to format this string, but tbh, it's pretty fine as is. Feel free to resolve.